Translations/French.csv

```diff
@@ -2216,6 +2216,8 @@ obj.materials.carbon,Carbone,,
 obj.materials.carbon.desc,Une matière première qui peut être extraite ou produite en brûlant quelque chose. Peut s'accumuler sous forme de résidu dans certains réacteurs.,,
 obj.materials.carbonfiber,Fibre de carbone,,
 obj.materials.carbonfiber.desc,Un matériau polyvalent très résistant et léger,,
+obj.materials.caoutchouc,Caoutchouc,,
```

Comment:
@rem02 When I was searching for Caotchouc in code, I found that French translation already uses this word for Rubber. Perhaps these two should somehow differ? Caotchouc is a "Natural Rubber", and "Rubber" could be "Synthetic Rubber",

Reply:
Indeed, for me, I understood the difference...
"Rubber" is the natural from a tree.
"rubber" is rubber transformed into rubber for tires, o-rings, elastic...

Reply:
I checked the in-game translation.
Indeed, this poses a problem of understanding.
In the game, the Caoutchouc comes from the tree and then it is used to make rubber ?
Otherwise, "Latex" is extracted from the tree which is then transformed into rubber.

Reply:
Latex is liquid, while Caoutchouc is a solid material in between latex and rubber. I did consider extracting latex from the rubber tree instead of caoutchouc, but a solid material is more convenient to handle.

Reply:
Yes, you are right. latex is liquid...
So Caoutchouc remains "Caoutchouc" and
I will change rubber to "Élastomère" in French.